code/modules/surgery/organs/autosurgeon.dm

```diff
@@ -69,6 +71,20 @@
 	else
 		return ..()
 
+/obj/item/autosurgeon/organ/attack(mob/living/target, mob/living/user, params)
+	add_fingerprint(user)
+	to_chat(user, span_notice("You begin to prepare to use [src] on [target]."))
```

Comment:
Replace this with a `balloon_alert_to_viewers`--the target should know they're being operated on